### PR TITLE
rollback/v0.12.1-v0.11.0

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

## Bug Fixes
- Reverting the changes introduced between v0.12.1 except Databricks compatibility. Please stay tuned for a future release that will integrate the v0.12.1 changes in a bug free release. ([#88](https://github.com/fivetran/dbt_jira/pull/88))

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide an explanation as to how the change is non-breaking below.)

This PR only rolls back the previous releases changes and will not be a breaking change

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature #83 
- [ ] No 

**How did you test the PR changes?** 

 

- [X] Buildkite  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🐛 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-joemarkiewicz** [comment]
For a list of changes between this branch and v0.11.0 release, see [this changelog](https://github.com/fivetran/dbt_jira/compare/v0.11.0...rollback/v0.12.1-v0.11.0).

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-joemarkiewicz Everything looks to go! Remember to regenerate docs.

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Include the following jira_source package version in your `packages.yml` file:
 ```yaml
```
Comment (fivetran-catfritz) [inline README.md:51]: It won't let me select the line, but just noticed line 49 should say `jira` instead of `jira_source`.

Comment (fivetran-joemarkiewicz) [reply README.md:51]: good call just updated.

**Pull request merged**